packages/code-du-travail-frontend/routes.js

```diff
@@ -44,18 +44,18 @@ module.exports = routes()
     pattern: "/modeles-de-courriers/:slug"
   })
 
-  // http://localhost:3000/kali/1930-nouvelle-convention-collective-nationale-des-metiers-de-la-transformation-des-gr
+  // http://localhost:3000/convention-collective/1930-nouvelle-convention-collective-nationale-des-metiers-de-la-transformation-des-gr
   .add({
-    name: "kali",
-    page: "kali",
-    pattern: "/kali/:slug"
+    name: "convention-collective",
```

Comment:
I wonder if we whould leave kali or not since it's the name of DB where data come from and 
the name convey some domain meaning

Reply:
In my opinion convention-collective convey much more meaning than kali. Since we will probably never have two DB sources to display a CC, and that, for the end user (no-expert), and in some extent me aswell, I have no idea what "kali" refers to in an URL. Naming it convention-collective everywhere seems a good option. We could call it kali in the API package ? But then we would create inconcistency.